parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -47,8 +47,9 @@ ERR_APT_DAILY_TIMEOUT=98 # Timeout waiting for apt daily updates
 ERR_APT_UPDATE_TIMEOUT=99 # Timeout waiting for apt-get update to complete
 ERR_CSE_PROVISION_SCRIPT_NOT_READY_TIMEOUT=100 # Timeout waiting for cloud-init to place this (!) script on the vm
 ERR_APT_DIST_UPGRADE_TIMEOUT=101 # Timeout waiting for apt-get dist-upgrade to complete
-ERR_CIS_HARDENING_ERROR=102 # Error applying CIS enforcement
```

Comment:
As we grow the CIS functions, we'll want more granular exit codes. Skipped forward to use the 110s range so we have a proper range for CIS errors.